### DISCUSSION TITLE
Synology/Docker/Portainer - Key Mismatch

### DISCUSSION THREAD

**Discussion by Will-WC2L** [rustdesk/rustdesk#10255] [General]
I'm a neophyte with the whole Synology/Docker/Portainer configuration. I got a few things that I wanted installed. The overall performance has been good.  I knew that there has been some feature and security updates.  This morning, I upgraded the OS and Packages to the latest versions. 

Now the RustDesk is giving me a "Key Mismatch" error! I have not been able to figure out how to access things either by command prompt or GUI. I have a few people that are relying on RustDesk (hobby related) and would like some guidance on how to fix it. I'm pretty sure I have app that is not happy, but lower on my priority list! 

I tried deleting the client pointer out of the client. I have uninstalled/reinstalled. I was looking for logs locally. 
Hopefully I can get this up ASAP!! 
TNX Will

**Comment by Will-WC2L** [comment]
I guess the upgrade enabled the required key. I didn't have one beforehand.
So after a bit of internet searches, I found the issue. Took even longer to find the correct file.
At least it is working.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/10163
